crates/chia-datalayer/src/lib.rs

```diff
@@ -1,42 +1,155 @@
-use std::collections::HashMap;
+// use std::collections::HashMap;
 
-type Index = usize;
-type Key = Vec<u8>;
+type TreeIndex = u32;
+// type Key = Vec<u8>;
+type Hash = [u8; 32];
+type KVId = Hash;
```

Comment:
Similarly to `TreeIndex`, you could make a newtype called `KvId([u8; 32])` instead of a type alias.

Note the acronym `Kv` is PascalCase too:
https://rust-lang.github.io/api-guidelines/naming.html

Reply:
case change incoming